File: src/kyu_7/all_unique/c/all_unique_test.c
#include <criterion/criterion.h>

bool has_unique_chars(const char *str);

Test(has_unique_chars, test_example) {
    cr_assert_not(has_unique_chars("  nAa"));
    cr_assert(has_unique_chars("abcde"));
    cr_assert_not(has_unique_chars("++-"));
    cr_assert(has_unique_chars("AaBbC"));
}

File: src/kyu_7/all_unique/python/all_unique_test.py
import codewars_test as test
from solution import has_unique_chars


@test.describe("Basic Examples")
def test_bacis():

    @test.it("should test characters uniqueness")
    def test_uniqueness():
        test.assert_equals(has_unique_chars("abcdef"), True)
        test.assert_equals(has_unique_chars("++-"), False)
        test.assert_equals(has_unique_chars("  nAa"), False)
